modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx

```diff
@@ -146,9 +172,77 @@ export default function EditNotificationTemplate({
 
 	const [templateTitle, setTemplateTitle] = useState<string>();
 
-	const [notificationType, setNotificationType] = useState<string>(
-		notificationTemplateType
-	);
+	const [rolesList, setRolesList] = useState<Role[]>([]);
+
+	const [userList, setUserList] = useState<User[]>([]);
+
+	const [multiSelectItems, setMultiSelectItems] = useState<Item[]>([]);
+
+	const [searchTerm, setSearchTerm] = useState('');
+
+	const getAccountRoles = async (searchTerm: string) => {
+		const apiURL = '/o/headless-admin-user/v1.0/accounts/0/account-roles';
+		const query = `${apiURL}?page=1&pageSize=10&sort=name:asc${
+			searchTerm ? `&filter=contains(name, '${searchTerm}')` : ''
+		}`;
+		const response = await fetch(query, {
+			headers: HEADERS,
+			method: 'GET',
+		});
+
+		const responseJSON = await response.json();
+
+		const roles = responseJSON.items.map((role: Role) => {
+			return {
+				label: role.displayName,
+				value: role.name,
+			};
+		});
```

Comment:
Hey @alinedoleron, I think we can unstructure the user, to just get the specific props you want here, like:

```ts

const roles = responseJSON.items.map(({displayName, name}: Role) => {
	return {
		label: displayName,
		value: name,
	};
});
```